### PR TITLE
add more LP stats

### PR DESCRIPTION
**Pull request by aymericdelab**

### CODE REVIEW

**Comment by aymericdelab** [comment]
no changes in contract

File: client/src/ui/components/bank/LiquidityResourceRow.tsx [hunk 1]
```diff
@@ -6,20 +6,35 @@ import Button from "@/ui/elements/Button";
 import { ResourceCost } from "@/ui/elements/ResourceCost";
 import { ResourceIcon } from "@/ui/elements/ResourceIcon";
 import { divideByPrecision, getEntityIdFromKeys } from "@/ui/utils/utils";
-import { ContractAddress, DONKEY_ENTITY_TYPE, ID, ResourcesIds, resources } from "@bibliothecadao/eternum";
+import {
+  ContractAddress,
+  DONKEY_ENTITY_TYPE,
+  ID,
+  RESOURCE_INPUTS_SCALED,
```
Comment (ponderingdemocritus) [inline client/src/ui/components/bank/LiquidityResourceRow.tsx:13]: this should be fetched from the config and not the SDK

File: client/src/ui/components/bank/LiquidityResourceRow.tsx [hunk 2]
```diff
@@ -6,20 +6,35 @@ import Button from "@/ui/elements/Button";
...
+  RESOURCE_OUTPUTS,
```
Comment (ponderingdemocritus) [inline client/src/ui/components/bank/LiquidityResourceRow.tsx:14]: same this this

**Review by ponderingdemocritus — COMMENTED** [review]
smol changes

**Pull request merged**